builtin/logical/ssh/path_issue_sign.go

```diff
@@ -53,9 +52,7 @@ type creationBundle struct {
 	Extensions      map[string]string
 }
 
-func (b *backend) pathSignIssueCertificateHelper(ctx context.Context, req *logical.Request, data *framework.FieldData, role *sshRole, publicKey ssh.PublicKey) (*logical.Response, error) {
-	// Note that these various functions always return "user errors" so we pass
-	// them as 4xx values
+func (b *backend) pathSignIssueCertificateHelper(sc *storageContext, req *logical.Request, data *framework.FieldData, role *sshRole, publicKey ssh.PublicKey) (*logical.Response, error) {
```

Comment:
In the response, it'd be great to inject the `issuer_ref` as well. I don't know if PKI does this, but given SSH certs are a little opaque, it'd be good to have a correlation ID.